service/controller/resources/vaultcrt/desired.go

```diff
@@ -31,9 +31,8 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		ObjectMeta: apismetav1.ObjectMeta{
 			Name: key.SecretName(customObject),
 			Annotations: map[string]string{
-				ConfigHashAnnotation:           hash,
-				UpdateTimestampAnnotation:      r.currentTimeFactory().In(time.UTC).Format(UpdateTimestampLayout),
-				VersionBundleVersionAnnotation: key.VersionBundleVersion(customObject),
```

Comment:
Shouldn't we keep some version label? We should make sure the operator is only working with their particular CRs it has authority over. Otherwise all kinds of operators modify all kinds of CRs.

Reply:
Those are annotations. But we should have `giantswarm.io/managed-by` label at least.

Reply:
That's done. PTALA